src/pages/Admin/VenueEventDetails.tsx

```diff
@@ -14,15 +14,17 @@ export interface VenueEventDetailsProps {
   setEditedEvent: Function | undefined;
   setShowCreateEventModal: Function;
   setShowDeleteEventModal: Function;
-  className: string;
+  className?: string;
+  isEditable?: boolean;
 }
 
 const VenueEventDetails = ({
   venueEvent,
   setEditedEvent,
   setShowCreateEventModal,
   setShowDeleteEventModal,
-  className,
+  className = "",
```

Comment:
className can probably be left undefined rather than defaulted to an empty string. At least as far as the DOM elements are concerned